hudi-spark-datasource/hudi-spark3.1.x/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/Spark31HoodieParquetFileFormat.scala

```diff
@@ -61,7 +61,7 @@ import java.net.URI
  *   <li>Schema on-read</li>
  * </ol>
  */
-class Spark312HoodieParquetFileFormat(private val shouldAppendPartitionValues: Boolean) extends ParquetFileFormat {
+class Spark31HoodieParquetFileFormat(private val shouldAppendPartitionValues: Boolean) extends ParquetFileFormat {
```

Comment:
the changes to 31 parquet format is purely refactoring right? looking ok but i would rather be super paranoid at this stage to introduce more diffs than needed.

Reply:
Aligning naming b/w Spark 3.2 and 3.1 to avoid confusion that this Spark specific version